INSTALL.md

```diff
@@ -99,35 +100,28 @@ thus does not need an include path.
 
 There are several BLAS implementations, depending on the OS and
 machine. To have reasonable performance, the BLAS library should be
-multithreaded.
-
-To check that the link flags are correct, and verify whether the
-implementation uses 32 or 64 bit integers, you can
-
-  `make test_blas`
-
-and run
-
-  `./misc/test_blas`
+multithreaded. On Intel platforms, MKL is the fastest BLAS implementation, 
+see flags above on how to enable it.
 
 
 Testing Faiss
 -------------
 
 A basic usage example is in
 
-  `demos/demo_ivfpq_indexing`
+  `demos/demo_ivfpq_indexing.cpp`
 
-which you can build by calling
-  `make demo_ivfpq_indexing`
+which you can build by calling 
```

Comment:
Looks like the command disappeared.

Reply:
build: `$ make -C build demo_ivfpq_indexing`
run: `$ ./build/demo_ivfpq_indexing`